src/Context/ToastContext.tsx

```diff
@@ -0,0 +1,24 @@
+import { createContext } from "react";
```

Comment:
We don't need authContext, we can put the ToastContainer in a top-level component such as the `App` or in  the `main`, and then use the functions provided by the package such as `toast.error("error Message")`  and `toast.success("success Message")`  and it provides a good default style .